### PR TITLE
Just added some CLI arguments

### PR DESCRIPTION
**Pull request by gp2112**
Hi!
I Just have added command options to allow the user define easily the number of random relays to test and the number of relays to scan (or until the end, if it's the case).

### CODE REVIEW

File: src/tor_relay_scanner/scanner.py [hunk 1]
```diff
@@ -4,11 +4,12 @@
 import random
 import sys
 import urllib.parse
-
+import argparse
 import requests
 
 TIMEOUT = 10.0
```
Comment (ValdikSS) [inline src/tor_relay_scanner/scanner.py:10]: Please remove this timeout, it's useless now

File: src/tor_relay_scanner/scanner.py [hunk 2]
```diff
@@ -164,4 +166,9 @@ async def main_async():
 
 
 def main():
-    asyncio.run(main_async())
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument('-n', type=int, dest='num_relays', nargs='?', default=30, help='Choose the number of relays to test.')
+    parser.add_argument('-g', '--goal', type=int, dest='working_relay_num_goal', nargs='?', default=5, help='Choose the max number of relays to scan.')
+    parser.add_argument('--timeout', type=float, nargs='?', default=10.0)
```
Comment (ValdikSS) [inline src/tor_relay_scanner/scanner.py:172]: Please write a text description for this option

**Review by ValdikSS — CHANGES_REQUESTED** [review]

File: src/tor_relay_scanner/scanner.py (hunk 2 above)
Comment (ValdikSS) [reply src/tor_relay_scanner/scanner.py:172]: num_relays → The number of concurrent relays tested
working_relay_num_goal → Test until at least this number of working relays are found
timeout → socket connection timeout

**Comment by gp2112** [comment]
@ValdikSS I've added the descriptions you had asked and removed TIMEOUT constant.

File: src/tor_relay_scanner/scanner.py [hunk 3]
```diff
@@ -37,10 +35,12 @@ async def connect(self):
             self.connection_status = False
             return (False, e)
 
-
 class TorRelayGrabber:
+    def __init__(self, timeout=10.0):
```
Comment (gp2112) [inline src/tor_relay_scanner/scanner.py:39]: just created the constructor to allow the timeout setting

**Comment by ValdikSS** [comment]
Looks great, thanks, merged!

**Pull request merged**